### PR TITLE
Update HTTPUpdate to add setFollowRedirects

### PR DESCRIPTION
**Pull request by DTTerastar**
This is a mirror of a change in esp8266 needed to update with a url that redirects.

### CODE REVIEW

**Comment by lbernstone** [comment]
Already merged
https://github.com/espressif/arduino-esp32/commit/ee88c42c3b5e87a2fc09303f9e82dd8172e99990

**Comment by DTTerastar** [comment]
I don't believe you looked at this close enough.  It is a change to HTTPUpdate NOT to HTTPClient.  It is not already merged and is a case where the esp8266 has better functionality than the esp32.

File: libraries/HTTPUpdate/src/HTTPUpdate.h [hunk 1]
```diff
@@ -89,6 +98,7 @@ class HTTPUpdate
     bool _rebootOnUpdate = true;
 private:
     int _httpClientTimeout;
+    followRedirects_t _followRedirects = HTTPC_DISABLE_FOLLOW_REDIRECTS;
```
Comment (me-no-dev) [inline libraries/HTTPUpdate/src/HTTPUpdate.h:101]: default value should be set in the class constructors instead.
```cpp
HTTPUpdate::HTTPUpdate(void)
        : _httpClientTimeout(8000), _followRedirects(HTTPC_DISABLE_FOLLOW_REDIRECTS), _ledPin(-1)
{
}
HTTPUpdate::HTTPUpdate(int httpClientTimeout)
        : _httpClientTimeout(httpClientTimeout), _followRedirects(HTTPC_DISABLE_FOLLOW_REDIRECTS), _ledPin(-1)
{
}
```

Comment (DTTerastar) [reply libraries/HTTPUpdate/src/HTTPUpdate.h]: Requested changes made.

**Comment by me-no-dev** [comment]
Thanks :)

**Pull request merged**